src/api/java/baritone/api/Settings.java

```diff
@@ -180,9 +180,14 @@
     /**
      * Blocks that Baritone will attempt to avoid (Used in avoidance)
      */
-    public final Setting<List<Block>> blocksToAvoid = new Setting<>(new ArrayList<>(
-            // Leave Empty by Default
-    ));
+    public final Setting<List<Block>> blocksToAvoid = new Setting<>(new ArrayList<>(Arrays.asList(
+            Blocks.FIRE,
+            Blocks.TRIPWIRE,
+            Blocks.WEB,
+            Blocks.END_PORTAL,
```

Comment:
why is end portal on the avoid list? when is this realistically going to be an issue?

Reply:
the avoid list is just copied from the hard coded values i removed.

Reply:
then we should still consider why it's actually in that list

Reply:
instant teleport while pathing could be bad to accidentally walk over/through